cusub_common/drivers/gazebo_drivers/scripts/dvl.py

```diff
@@ -40,4 +40,4 @@ def run(self):
     try:
         DVL_REMAP.run()
     except rospy.ROSInterruptException:
-        rospy.loginfo("Gazebo DVL driver died!")
+        rospy.logerr("Gazebo DVL driver died!")
```

Comment:
Is there a specific reason for this one being left as logerr? versus cuprint(err=True)?